climada/hazard/tc_tracks.py

```diff
@@ -387,36 +402,52 @@ def read_ibtracs_netcdf(self, provider=None, rescale_windspeeds=True, storm_id=N
         if estimate_missing and not rescale_windspeeds:
             LOGGER.warning(
                 "Using `estimate_missing` without `rescale_windspeeds` is strongly discouraged!")
-        self.data = list()
-        fn_nc = SYSTEM_DIR.joinpath(file_name)
-        if not fn_nc.is_file():
+
+        ibtracs_path = SYSTEM_DIR.joinpath(file_name)
+        if not ibtracs_path.is_file():
             try:
                 download_ftp(f'{IBTRACS_URL}/{IBTRACS_FILE}', IBTRACS_FILE)
-                shutil.move(IBTRACS_FILE, fn_nc)
+                shutil.move(IBTRACS_FILE, ibtracs_path)
             except ValueError as err:
                 raise ValueError(
                     f'Error while downloading {IBTRACS_URL}. Try to download it manually and '
-                    f'put the file in {SYSTEM_DIR}') from err
+                    f'put the file in {ibtracs_path}') from err
+
+        ibtracs_ds = xr.open_dataset(ibtracs_path)
+        ibtracs_date = ibtracs_ds.attrs["date_created"]
+        LOGGER.info(f"The IBTrACS data set dates from {ibtracs_date}")
+        if 180 < (np.datetime64('today') - np.datetime64(ibtracs_date)) / np.timedelta64(1, 'D'):
+            LOGGER.warning("The cached IBTrACS data set is older than 180 days. "
+                           "Very likely, a more recent version is available. "
+                           f"Consider manually removing the file {ibtracs_path} and re-running "
+                           "this function, which will download the most recent version of the "
+                           "IBTrACS data set from the official URL.")
```

Comment:
I found that users tend to use data for very recent years even though this kind of data is very often subject to changes in IBTrACS updates. But even for older data, there might be interesting updates from time to time. CLIMADA currently caches the IBTrACS data set and uses this version indefinitely. That's fine for reproducibility and stability of code, but will very often go unnoticed by the user. This update is to inform the user about this fact so that they may decide to manually switch to a more recent version of the data.

Reply:
This is an excellent point!

Reply:
I like, too!